### DISCUSSION TITLE
Downloading of translations is automatically converting `>` into `&gt;`

### DISCUSSION THREAD

**Discussion by mdobranic** [crowdin/github-action#253] [Q&A] [answered]
Hello,

In my English resx file I have added `> {amount}`.
When translated in Crowdin UI, it was set the same in the secondary language as `> {amount}` as the amount in brackets is being replaced with numeric value.
However, when I run the crowdin github action to download translations, I am getting: `&gt; {amount}`

Is this expected?
The problem is not in the application handling, it will display `>` and `&gt;` the same way, but now I have in one resix file `>` and in translated resx `&gt;`.

Thanks!

**Comment by andrii-bodnar** [comment, marked as answer]
Hi @mdobranic, thanks for reporting this! This issue is more related to the Crowdin file export process, not the GitHub action itself. 

Please [contact](https://crowdin.com/contacts) our Customer Success team for further investigation.
